src/main/java/com/ldtteam/blockui/views/BOWindow.java

```diff
@@ -235,8 +244,7 @@ public boolean onUnhandledKeyTyped(final int ch, final int key)
      */
     public void close()
     {
-        screen.removed();
-        this.mc.player.closeContainer();
+        Minecraft.getInstance().popGuiLayer();
     }
```

Comment:
there are a lot of significant changes, did you test this with minecolonies to make sure it doesnt fully break? Can just build a jar and test outside of dev as well

Reply:
These are safe 99.9%, the forge layering calls screen closing/removal methods, the close container call was bullshit anyway - we don't support inventories/slots, regarding stacked UIs we have today they are not aware of layering either, so this shouldn't/won't affect them either